src/classes/Carts/UserCart.ts

```diff
@@ -513,7 +514,11 @@ export default class UserCart extends Cart {
                         skuCount.mostCanTrade
                     )}`;
 
-                    this.addTheirItem(sku, skuCount.mostCanTrade);
+                    // Add the amount we can trade and substract added amount from command (if exist)
+                    this.addTheirItem(
+                        sku,
+                        this.their[sku] ? skuCount.mostCanTrade - this.their[sku] : skuCount.mostCanTrade
+                    );
```

Comment:
revert back to skuCount.mostCanTrade as I don't see any evidence of trades where the text says "I can't buy more" only "I can only sell more" which makes me think ony line 437 needs to be updated to match line 516.